src/modules/UI/components/ABAlert/reducer.js

```diff
@@ -19,24 +29,27 @@ const view = (state = false, action: Action) => {
   }
 }
 
-const syntax = (state = {}, action: Action) => {
+const syntax = (state = {}, action: Action): $PropertyType<ABAlertState, 'syntax'> => {
   switch (action.type) {
     case 'OPEN_AB_ALERT': {
+      if (action.data == null) throw new TypeError('Invalid action')
       return action.data
     }
 
     case 'CLOSE_AB_ALERT': {
+      // $FlowFixMe
```

Comment:
Bogus return type.

Reply:
@wolverineks Are we using the `route` property on ABAlert right now or no?